pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -53,7 +54,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-const finalizer = "foreground-deletion"
+const (
+	finalizer   = "foreground-deletion"
+	nodeCfgName = "cluster"
```

Comment:
will this always be the Node config name ?

Reply:
Yes, the node config object named `cluster` is the only custom resource that gets created in the cluster.